modules/machineconfig-garbage-collect-viewing.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineconfig-garbage-collect-viewing_{context}"]
+= Viewing rendered machine configs
+
+You can view a list of rendered machine configs by using the `oc adm prune renderedmachineconfigs` command with the `list` subcommand. For example, the command in the following procedure would list all rendered machine configs for the `worker` machine config pool:   
+
+.Procedure
+
+* List the rendered machine configs:
++
+[source,terminal]
+----
+$ oc adm prune renderedmachineconfigs list --in-use --pool-name=worker
+----
++
+.Example output
+[source,terminal]
+----
+                                     worker
+rendered-worker-b335d3cdd1f83d0fce1657389a09e87d -- 2024-05-16 17:21:06 +0000 UTC (Currently in use: false)
+rendered-worker-37f7b2d99f64fbbd91d8bf3f0b50ad83 -- 2024-05-16 18:21:35 +0000 UTC (Currently in use: true)
+----
++
```

Comment:
This is not the output that corresponds to the previous command `oc adm prune renderedmachineconfigs list --in-use --pool-name=worker`


The output is:

```
$ oc adm prune renderedmachineconfigs list --in-use --pool-name=worker
worker
status: rendered-worker-ae115e2b5e6ae05e0e6e5d62c7d0dd81
spec: rendered-worker-ae115e2b5e6ae05e0e6e5d62c7d0dd81

```